worlds/ladx/__init__.py

```diff
@@ -474,7 +475,8 @@ def generate_output(self, output_directory: str):
             rnd=self.multiworld.per_slot_randoms[self.player],
             player_name=name_for_rom,
             player_names=all_names,
-            player_id = self.player)
+            player_id = self.player,

```

Comment:
all of them without spaces and then randomly one of them with spaces  is killing me.